landscape/categories.md

```diff
@@ -0,0 +1,56 @@
+## Proposed Security Landscape Structure
+In this section of the document, we outline our proposal for the structure of the Security Landscape.
+
+The bullets in **bold** are the primary categories in the Security Landscape and the bullets in _italics_ are the
+sub-categories. The remaining text is a description of each sub-category.
+
+- **App definition and development**
+  - _Static Code Analysis_
+    - Inspecting code for OWASP vulnerabilities, pen testing, etc
+  - _Dependency analysis_
+    - Checking OS for vulnerabilities (updates available? Reports from https://www.us-cert.gov/ etc), image scanning
+    - Check for vulnerabilities in dependent libraries
+    - Check for maintenance of dependent projects => increased potential for vulnerabilities
+  - _Pipeline security testing_
+    - Tools that facilitate automated security testing in pipelines; eg for functional security tests of authn and authz,
```

Comment:
Should this category be renamed to be clearer about what it contains (right now, just tools to automate security testing)? Should we enlarge it to also include other tools to secure CI/CD pipelines? How does supply chain management fit in?